impeller/display_list/display_list_dispatcher.cc

```diff
@@ -701,7 +701,9 @@ void DisplayListDispatcher::drawPaint() {
 // |flutter::Dispatcher|
 void DisplayListDispatcher::drawLine(const SkPoint& p0, const SkPoint& p1) {
   auto path = PathBuilder{}.AddLine(ToPoint(p0), ToPoint(p1)).TakePath();
-  canvas_.DrawPath(std::move(path), paint_);
+  Paint paint = paint_;
+  paint.style = Paint::Style::kStroke;
+  canvas_.DrawPath(std::move(path), std::move(paint));
```

Comment:
Yeah, either that or have a "force stroking" flag to DrawPath.

Another primitive that forces stroking is DrawPoints.